nitro-protocol/test/contracts/libraries/Consensus/Consensus.test.ts

```diff
@@ -86,7 +86,9 @@ describe('requireConsensus', () => {
       if (reason) {
         await expectRevert(() => Consensus.requireConsensus(fixedPart, proof, candidate));
       } else {
-        await expectSucceedWithNoReturnValues(() => Consensus.requireConsensus(fixedPart, proof, candidate));
+        await expectSucceedWithNoReturnValues(() =>
```

Comment:
These typescript changes were prompted by the CI failure https://github.com/statechannels/go-nitro/actions/runs/5423048165/jobs/9860465189

A little confusing for this to fail now - eslint is a devDependency pinned at 7.17.0., which was configured ~15 months ago. Running the same version on my own machine produces the same flagged issues